app/models/talk.rb

```diff
@@ -133,4 +136,18 @@ def related_talks(limit: 6)
   def transcript
     enhanced_transcript.presence || raw_transcript
   end
+
+  def update_from_yml_metadata
+    self.title = yml_metadata["title"]
+    self.description = yml_metadata["description"]
+    self.date = yml_metadata["published_at"]
```

Comment:
You probbaly also want to double check the speakers here. I noticed that a bunch of speakers had some typos or other small details differently that could cause a mismatch

Reply:
Good call. I will have a look just to make sure we don’t create duplicate speakers when we correct a typo

Reply:
Opened #120 which might help with this